General.ruleset

```diff
@@ -0,0 +1,110 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<RuleSet Name="General C# rules" Description="" ToolsVersion="16.0">
```

Comment:
What should we do if rules from multiple tools overlap? E.g. this one: https://rules.sonarsource.com/csharp/RSPEC-3963 comes from Sonar but also from .editorconfig so there's a warning even if it's suppressed for .editorconfig:

![image](https://user-images.githubusercontent.com/1976647/88116938-1b43b880-cbba-11ea-8866-2e31d0ee5a92.png)

Should we just use Sonar?

Reply:
I would leave Sonar one. But you decide what to do with `.editorconfig` setting.

Reply:
We don't want to use two rules for the same thing so please disable this in .editorconfig with the comment that we're using RSPEC-3963 instead. In subsequent cases please do the same.

Reply:
Disabled.